src/commands/view/OpenManagers.js

```diff
@@ -0,0 +1,116 @@
+import Backbone from 'backbone';
+const StyleManager = require('style_manager');
+const $ = Backbone.$;
+
+module.exports = {
+  run(em, sender) {
+    this.sender = sender;
+    this.em = em;
+    if (!this.$tmCn || !this.smCn) {
+      this.prepareTm();
+      this.prepareSm();
+      this.listenTo(this.target, 'component:toggled', this.toggleManagers);
+    }
+    this.toggleManagers();
+  },
+  prepareTm() {
+    const em = this.em;
+    const config = em.Config;
+    const pfx = config.stylePrefix;
+    const tm = em.TraitManager;
+    var panelC;
+
+    if (!this.$tmCn) {
+      var tmView = tm.getTraitsViewer();
+      var confTm = tm.getConfig();
+      this.$tmCn = $('<div></div>');
+      this.$tmCn2 = $('<div></div>');
+      this.$tmCn.append(this.$tmCn2);
+      this.$tmHeader = $('<div>').append(
+        `<div class="${confTm.stylePrefix}header">${confTm.textNoElement}</div>`
+      );
+      this.$tmCn.append(this.$tmHeader);
+      this.$tmCn2.append(
+        `<div class="${pfx}traits-label">${confTm.labelContainer}</div>`
+      );
+      this.$tmCn2.append(tmView.render().el);
+      var panels = em.Panels;
+
+      if (!panels.getPanel('views-container'))
+        panelC = panels.addPanel({ id: 'views-container' });
+      else panelC = panels.getPanel('views-container');
+
+      panelC
+        .set('appendContent', this.$tmCn.get(0))
+        .trigger('change:appendContent');
+
+      this.target = em.getModel();
+      this.listenTo(this.target, 'component:toggled', this.toggleTm);
+    }
+  },
+
+  prepareSm() {
+    if (!this.$smCn) {
+      const em = this.em;
+      var config = em.getConfig(),
+        panels = em.Panels;
+      // Main container
+      this.$smCn = $('<div></div>');
```

Comment:
I wonder how he chose this naming convention? It seems like he could have just added a couple of characters and named them what they are.
eg
`this.$styleMainContainer` vs `this.$smCn`

Reply:
Artur is the code golfing king :crown: :golfing_man: